test/integ/pandas/test_arrow_pandas.py

```diff
@@ -1037,3 +1037,29 @@ def test_pandas_telemetry(
         assert occurence == 1
 
         finish(conn, table)
+
+
+@pytest.mark.parametrize("result_format", ["pandas", "arrow"])
+def test_batch_to_pandas_arrow(conn_cnx, result_format):
+    rowcount = 10
+    with conn_cnx() as cnx:
+        cursor = cnx.cursor()
+        cursor.execute(SQL_ENABLE_ARROW)
```

Comment:
I would prefer embedding these into the connection object, like you do it in your other PR, but this is more of a nitpick than a hard requirement